rules/repositories.bzl

```diff
@@ -91,33 +123,80 @@ def rules_ios_dependencies(
         )
     else:
         _maybe(
-            github_repo,
+            http_archive,
             name = "build_bazel_rules_swift",
-            project = "bazelbuild",
-            ref = "17e20f7edf27e647f1b45f11ed75d51c17820c3b",
-            repo = "rules_swift",
-            sha256 = "d50c2cb6f1c2c30cf44a8ea60469cd399f7458061169bde76a177b63d6b74330",
+            sha256 = "3a595a64afdcaf65b74b794661556318041466d727e175fa8ce20bdf1bb84ba0",
+            url = "https://github.com/bazelbuild/rules_swift/releases/download/1.10.0/rules_swift.1.10.0.tar.gz",
         )
 
         _maybe(
-            github_repo,
+            http_archive,
             name = "build_bazel_rules_apple",
-            ref = "915ac30a9fa1fd3809599a5ab90fa1c6640fe8dc",
-            project = "bazelbuild",
-            repo = "rules_apple",
-            sha256 = "0204016496a39d5c70247650e098905d129f25347c7e1f019f838ca74252ce2d",
+            sha256 = "8ac4c7997d863f3c4347ba996e831b5ec8f7af885ee8d4fe36f1c3c8f0092b2c",
+            url = "https://github.com/bazelbuild/rules_apple/releases/download/2.5.0/rules_apple.2.5.0.tar.gz",
         )
 
     _maybe(
         http_archive,
         name = "bazel_skylib",
+        sha256 = "b8a1527901774180afc798aeb28c4634bdccf19c4d98e7bdd1ce79d1fe9aaad7",
+        urls = [
+            "https://mirror.bazel.build/github.com/bazelbuild/bazel-skylib/releases/download/1.4.1/bazel-skylib-1.4.1.tar.gz",
+            "https://github.com/bazelbuild/bazel-skylib/releases/download/1.4.1/bazel-skylib-1.4.1.tar.gz",
+        ],
+    )
+
+def _rules_ios_bzlmod_dev_dependencies():
+    """
+    Fetches repositories that are development dependencies of `rules_ios` and available via bzlmod.
+
+    These are only included when using WORKSPACE, when using bzlmod they're loaded in MODULE.bazel
+    """
+
+    _maybe(
+        http_archive,
+        name = "rules_pkg",
         urls = [
-            "https://github.com/bazelbuild/bazel-skylib/releases/download/1.1.1/bazel-skylib-1.1.1.tar.gz",
-            "https://mirror.bazel.build/github.com/bazelbuild/bazel-skylib/releases/download/1.1.1/bazel-skylib-1.1.1.tar.gz",
+            "https://mirror.bazel.build/github.com/bazelbuild/rules_pkg/releases/download/0.9.1/rules_pkg-0.9.1.tar.gz",
+            "https://github.com/bazelbuild/rules_pkg/releases/download/0.9.1/rules_pkg-0.9.1.tar.gz",
         ],
-        sha256 = "c6966ec828da198c5d9adbaa94c05e3a1c7f21bd012a0b29ba8ddbccb2c93b0d",
+        sha256 = "8f9ee2dc10c1ae514ee599a8b42ed99fa262b757058f65ad3c384289ff70c4b8",
     )
 
+    _maybe(
+        http_archive,
+        name = "buildifier_prebuilt",
+        sha256 = "e46c16180bc49487bfd0f1ffa7345364718c57334fa0b5b67cb5f27eba10f309",
+        strip_prefix = "buildifier-prebuilt-6.1.0",
+        urls = [
+            "http://github.com/keith/buildifier-prebuilt/archive/6.1.0.tar.gz",
```

Comment:
In the lines you've removed we already pulled the offical sources:
https://github.com/bazel-ios/rules_ios/pull/703/files#diff-5493ff8e9397811510e780de47c57abb70137f1afe85d1519130dc3679d60ce5L66

Unless there is blocking reason to change this, I'd prefer to keep that. Is there a reason you just update the URL to use a newer version of the official cut?

Reply:
The previous WORKSPACE used an `http_file` download, I thought it made sense to use the `buildifier-prebuilt` which supports switching based on platform and supports bzlmod